packages/lib/src/spatial-navigation/components/FocusableView.tsx

```diff
@@ -44,13 +46,54 @@ export const SpatialNavigationFocusableView = forwardRef<SpatialNavigationNodeRe
 
     return (
       <SpatialNavigationNode isFocusable {...props} ref={nodeRef}>
-        {({ isFocused, isActive }) => (
-          <View style={style} {...viewProps} {...webProps}>
-            {typeof children === 'function' ? children({ isFocused, isActive }) : children}
-          </View>
-        )}
+        {({ isFocused, isActive }) => {
+          return (
+            <FocusableView
+              viewProps={viewProps}
+              webProps={webProps}
+              style={style}
+              isActive={isActive}
+              isFocused={isFocused}
+              ref={viewRef}
+            >
+              {children}
+            </FocusableView>
+          );
+        }}
       </SpatialNavigationNode>
     );
   },
 );
 SpatialNavigationFocusableView.displayName = 'SpatialNavigationFocusableView';
+
+type FocusableViewProps = {
```

Comment:
Shouldn't you use the above props to define these children props?
Also, you should comment that this is only to access the SpatialNavigationNode context that you split it up

Reply:
done :)